appveyor.yml

```diff
@@ -25,24 +25,10 @@ skip_commits:
     - .templates/**/*
 
 build_script:
-  - set PATH=C:\google-cloud-sdk\bin;%PATH%
```

Comment:
The Appveyor E2E testing gives us a indication that the tests work on Windows machine. I am not sure if we should remove it as @damithc has a Windows computer and the pre-deployment test is run by him. Does the driver behave the same way on both Windows and Linux?

Reply:
We're going to have Saucelabs to cover that and so much more.